lang/ko.json

```
@@ -429,7 +429,7 @@
       "events": {
         "complete": "탐험 성공!\n{LOCATION}를 방문하여, {EXP}를 획득했습니다",
         "freeComplete": "탐험 성공!\n{LOCATION}를 방문하였습니다.",
```

```suggestion
        "freeComplete": "탐험 성공!\n{LOCATION}를 방문하였습니다",
```
Should this period be removed for consistency?